tests/taglib/utils.rs

```diff
@@ -0,0 +1,94 @@
+#[macro_export]
```

Comment:
The tests could be moved into `tests/files` so you don't have to copy this file out.

Reply:
actually I am planing on doing all taglib tests, that's why they are in this folder.